x/ibc-rate-limit/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-
+resolver = "2"
```

Comment:
[it is said in the docs](https://doc.rust-lang.org/cargo/reference/resolver.html#resolver-versions) that the resolver is set to version 2 by default if the edition is 2021 or newer. the target branch [defines the edition as 2021](https://github.com/neutron-org/neutron/blob/051a12f0a395be62a8f5c8b6b693f4614a1098b6/x/ibc-rate-limit/contracts/rate-limiter/Cargo.toml#L5), so this line is not necessary. please correct me if I'm wrong

Reply:
You are most definitely correct, I have pushed a commit which changes this.